### PR TITLE
Signin/petros

### PR DESCRIPTION
**Pull request by chrisxadvocatus**

### CODE REVIEW

File: .env [hunk 1]
```diff
@@ -0,0 +1,2 @@
+MONGO_URI=mongodb+srv://admin:Breakthrough12321@db.v9yeuvo.mongodb.net/db?retryWrites=true&w=majority
+PORT=3000
```
Comment (chrisxadvocatus) [inline .env:2]: your credentials

File: server/controllers/UserControllers.js [hunk 2]
```diff
@@ -2,17 +2,102 @@ import User from '../models/UserModel.js'
 import jwt from 'jsonwebtoken'
```
Comment (chrisxadvocatus) [inline server/controllers/UserControllers.js:2]: are we still using jwt?

**Comment by chrisxadvocatus** [comment]
please pull from dev and resolve conflicts

**Comment by anthgrim** [comment]
Awaiting changes

**Review by anthgrim — APPROVED** [review]
Great job :)

**Comment by anthgrim** [comment]
Merging to main (dev)

**Pull request merged**